integration/commands_administration_test.go

```diff
@@ -572,6 +572,12 @@ func TestCommandsAdministrationBuildInfo(t *testing.T) {
 	assert.Equal(t, int32(16777216), must.NotFail(doc.Get("maxBsonObjectSize")))
 	_, ok = must.NotFail(doc.Get("buildEnvironment")).(*types.Document)
 	assert.True(t, ok)
+
+	compatibility, ok := must.NotFail(doc.Get("compatibility")).(*types.Document)
+	assert.True(t, ok)
+	assert.NotEmpty(t, compatibility)
+	aggregationStages := must.NotFail(compatibility.Get("aggregationStages")).(*types.Array)
+	assert.NotEmpty(t, aggregationStages)
```

Comment:
I think we should extract that part into its own test that would use `setup.SkipForMongoDB`. See this test for example: https://github.com/FerretDB/FerretDB/blob/d0022fb2c305e5ab44976cc28dd967a1fafc11a5/integration/basic_test.go#L519-L520